docs/runtimes/application.ipynb

```diff
@@ -167,7 +167,9 @@
   {
    "cell_type": "markdown",
    "metadata": {},
-   "source": "An API gateway can also be created manually. See the instructions [here](../concepts/nuclio-real-time-functions.ipynb)."
+   "source": [
+    "An API gateway can also be created manually. See the instructions [here](../concepts/nuclio-real-time-functions.ipynb)."
```

Comment:
"An API gateway can also be created manually. See the [instructions](../concepts/nuclio-real-time-functions.ipynb#api-gateway)."